src/mainboard/amd/romecrb/src/main.rs

```diff
@@ -475,8 +451,7 @@ pub extern "C" fn _start(fdt_address: usize) -> ! {
 #[panic_handler]
 fn panic(info: &PanicInfo) -> ! {
     // Assume that uart0.init() has already been called before the panic.
-    let io = &mut IOPort;
-    let uart0 = &mut I8250::new(0x3f8, 0, io);
+    let uart0 = &mut I8250::new(0x3f8, 0, IOPort {});
```

Comment:
this kind of points out another problem: the IoPort{} probably ought to have a base, range, and stride.

e.g.
IoPort.New(0x3f8, 8, 1) -> 3f8 for 8 bytes with a stride of 1 (meaning use inb/outb)

for cf8/cfc:
ioport.New(0xcf8, 8, 4);

our ioport abstraction is lacking.